src/main/java/com/regnosys/testing/testpack/TestPackFunctionRunnerProviderImpl.java

```diff
@@ -54,11 +74,11 @@ public TestPackFunctionRunner create(PipelineModel.Transform transform, Pipeline
         ObjectWriter outputObjectWriter = getObjectWriter(outputSerialisation).orElse(JSON_OBJECT_WRITER);
         // XSD validation
         Validator xsdValidator = getXsdValidator(outputType, outputSchemaMap);
-        return createTestPackFunctionRunner(outputType, inputType, injector, outputObjectWriter, xsdValidator);
+        return createTestPackFunctionRunner(toClass(transform.getFunction()), inputType, injector, outputObjectWriter, xsdValidator);
     }
 
-    private <IN extends RosettaModelObject> TestPackFunctionRunner createTestPackFunctionRunner(Class<?> outputType, Class<IN> inputType, Injector injector, ObjectWriter outputObjectWriter, Validator xsdValidator) {
-        Function<IN, RosettaModelObject> transformFunction = getTransformFunction(outputType, inputType, injector);
```

Comment:
Output type (e.g. Document) is not a transform function, it's just a class

Reply:
You still need to pass in a the function class, because you still need to create a function instance so you can run the pipeline.. i.e. convert the input into an output